tox.ini

```
@@ -90,6 +93,8 @@ commands =
          ext/opentelemetry-ext-http-requests/tests/ \
          ext/opentelemetry-ext-jaeger/src/opentelemetry \
          ext/opentelemetry-ext-jaeger/tests/ \
+         ext/opentelemetry-ext-pymongo/src/ \
```

```suggestion
         ext/opentelemetry-ext-pymongo/src/opentelemetry \
```

I usually get errors when using just `src/` here. I wonder why this works without that.